src/core/frame/to_arrow.cc

```diff
@@ -166,6 +166,7 @@ std::unique_ptr<dt::OArrowSchema> Column::to_arrow_schema() const {
     case dt::SType::INT64:   (*osch)->format = "l"; break;
     case dt::SType::FLOAT32: (*osch)->format = "f"; break;
     case dt::SType::FLOAT64: (*osch)->format = "g"; break;
+    case dt::SType::DATE32:  (*osch)->format = "tdD"; break;
```

Comment:
May be we add a comment as to what are all these "n", "b", "tdD", etc. identifiers?

Reply:
ok